source/manual/data-gov-uk-2nd-line.html.md

```diff
@@ -226,6 +226,42 @@ Publishers upload their organograms as a Excel (XLS) file that contains macros.
 
 It is possible to access analytics for datasets. If a user requests analytics for datasets, we can provide them with access to an analytics dashboard. Assign tickets like this to Martin Lugton or another member of the Platform Health product team.
 
+## Differences in CKAN 2.9 vs 2.7/2.8
+
+If a publisher is attempting to access data directly via the use of the CKAN API endpoints they might encounter problems processing the response:
+
+- V1 of the API endpoint has been dropped - https://github.com/ckan/ckan/blob/ckan-2.9.2/CHANGELOG.rst#v280-2018-05-09
```

Comment:
Looks like the [public documentation](https://guidance.data.gov.uk/get_data/api_documentation/#api-documentation) for the API still points to the CKAN 2.7 documentation.